docs/csharp/tour-of-csharp/arrays.md

```diff
@@ -0,0 +1,86 @@
+---
+title: C# Arrays | A tour of the C# language
+description: Arrays are the most basic collection type in the C# langauge
+keywords: .NET, csharp, array, collection
+author: BillWagner
+manager: wpickett
+ms.date: 08/10/2016
+ms.topic: article
+ms.prod: visual-studio-dev-14
+ms.technology: devlang-csharp
+ms.devlang: csharp
+ms.assetid: a440704c-9e88-4c75-97dd-bfe30ca0fb97
+---
+
+# Arrays
+
+An ***array*** is a data structure that contains a number of variables that are accessed through computed indices. The variables contained in an array, also called the ***elements*** of the array, are all of the same type, and this type is called the ***element type*** of the array.
+
+Array types are reference types, and the declaration of an array variable simply sets aside space for a reference to an array instance. Actual array instances are created dynamically at run-time using the new operator. The new operation specifies the ***length*** of the new array instance, which is then fixed for the lifetime of the instance. The indices of the elements of an array range from `0` to `Length - 1`. The `new` operator automatically initializes the elements of an array to their default value, which, for example, is zero for all numeric types and `null` for all reference types.
```

Comment:
from our old guidelines, run-time here should be not hyphened and spelled like this: run time.
Use run time to indicate the period of time during which a program is running. 
Use runtime as an abbreviation for common language runtime or, in a more generic sense, to indicate an active program during run time. 
Use run-time as an adjective.